src/diff/props.js

```diff
@@ -98,7 +98,7 @@ export function setProperty(dom, name, value, oldValue, isSvg) {
 			);
 		}
 	} else {
-		if (isSvg) {
+		if (namespace == 'http://www.w3.org/2000/svg') {
```

Comment:
Is this intentionally a double equals?

Reply:
Shaves 1b from `preact.module.js.br` :p

Should be safe